src/Forms/FormField.php

```diff
@@ -355,14 +356,18 @@ protected function setupDefaultClasses()
     }
 
     /**
-     * Return a link to this field.
+     * Return a link to this field
      *
      * @param string $action
-     *
      * @return string
+     * @throws LogicException If no form is set yet
      */
     public function Link($action = null)
     {
+        if (!$this->form) {
+            throw new LogicException('No form is set yet');
```

Comment:
Happy with this, can you just add a little more detail to the message? Something like

`Field must be associated with a form to call Link(). Please use $field->setForm($form);`